scripts/build.rs

```diff
@@ -43,7 +43,7 @@ fn cmake_build() {
 
     // Disable parallel builds on Windows, as they seems to break manifest builds.
     if cfg!(windows) {
-        config.define("CMAKE_BUILD_PARALLEL_LEVEL", "1");
+        env::set_var("CMAKE_BUILD_PARALLEL_LEVEL", "1");
```

Comment:
Shouldn't this be set in `cmake::env`?
It is possible (not sure) that the cmake environment variable default to the system environment variables, but conceptually, I think they are different.

- https://docs.rs/cmake/latest/cmake/struct.Config.html#method.env
- https://cmake.org/cmake/help/latest/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html
- https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#cmake-language-environment-variables

Reply:
```suggestion
        config.env("CMAKE_BUILD_PARALLEL_LEVEL", "1");
```

Reply:
I can try it. The build succeeded with what I have. Let's see if does with this change...

Reply:
This didn't work.